content/integrations/slack.md

```diff
@@ -1,13 +1,32 @@
 ---
 title: Datadog-Slack Integration
 integration_title: Slack
+git_integration_title: slack
 kind: integration
-doclevel: basic
+newhlevel: true
 ---
 
+# Overview
 
+Connect Slack to Datadog in order to help your team collaborate by:
 
-* In your Slack account Administration page, go to the Integration tab and choose Datadog.
-* Copy paste the service hook into the field below.
-* Add the channels you want to be able to post to.
-* Tick the checkbox if you want to be notified for every comment, otherwise you will need to use the @slack-channel_name syntax.
+* Sharing graphs with your colleagues in your team's channels.
+* Receiving alerts and notifications from Datadog with in Slack.
+
+
+# Installation
+
+Slack is installed via its [integration tile](https://app.datadoghq.com/account/settings#integrations/slack) in the Datadog application.
```

Comment:
Prefer active voice: "Install Slack integration via", not "Slack is installed via"